dynamodb/src/main/scala/awscala/dynamodbv2/Table.scala

```diff
@@ -106,10 +135,14 @@ case class Table(
     val methodNames = obj.getClass.getDeclaredMethods
       .filter(m => Modifier.isPublic(m.getModifiers))
       .filterNot(m => Modifier.isStatic(m.getModifiers))
-      .filterNot(m => m.getParameterTypes.size > 0)
+      .filterNot(m => m.getParameterTypes.length > 0)
       .map(_.getName)
 
     methodNames.filter(m => fieldNames.contains(m))
+      .map(getterName => {
+        val value = obj.getClass.getDeclaredMethod(getterName).invoke(obj)
+        getterName -> value
+      }).toList
```

Comment:
Why did you add toList here? I don't see the necessity to use List over Seq here.

Reply:
fixed